.travis.yml

```diff
@@ -1,7 +1,15 @@
 language: node_js
 sudo: false
 node_js:
-  - '0.12'
+  - '4.2.2'
+
+addons:
+  apt:
+    sources:
+    - ubuntu-toolchain-r-test
+    packages:
+    - gcc-4.8
```

Comment:
this compiler dependency is introduced for `selenium-webdriver` requires C++11 supported compilers with node.js 4.